dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -101,3 +101,11 @@ contextRootForFormAuthenticationMechanism.desc=Specifies the context root of a f
 
 basicAuthenticationMechanismRealmName=Realm name of the Java EE 8 basic authentication
 basicAuthenticationMechanismRealmName.desc=Specifies a realm name for the Java EE 8 Security HTTP basic authentication. This value is applicable when overrideHttpAuthMethod is set to BASIC, or if overrideHttpAuthMethod is set to CLIENT_CERT and allowAuthenticationFailOverToAuthMethod is set to BASIC. The default value is defaultRealm.
+
+# The word SameSite should not be translated
+sameSiteCookie=Add the SameSite attribute to the SSO cookie
+sameSiteCookie.desc=Specifies the SameSite attribute value to use for the SSO cookie.
```

Comment:
Specifies the SameSite attribute value to use for the SSO cookie.

**change to**
Specify the SameSite attribute value to use for the SSO cookie.